src/main/asciidoc/index.adoc

```diff
@@ -913,8 +913,10 @@ vertx run groovy:io.vertx.example.MyGroovyVerticle
 
 The `vertx run` command can take a few optional parameters, they are:
 
- * `-conf <config_file>` - Provides some configuration to the verticle. `config_file` is the name of a text file
- containing a JSON object that represents the configuration for the verticle. This is optional.
+ * `-vertx-options <options>` - Provides the Vert.x options. `options` is the name of a text file
```

Comment:
Do be renamed once the name is finalized.

Reply:
I would not say "txt" file but JSON file. No?

Reply:
Done